### PR TITLE
Include the orchestration package instructions in the README.md file

### PR DESCRIPTION
**Pull request by alek-kam-robotec-ai**
Signed-off-by: alek-kam-robotec-ai <aleksander.kaminski@robotec.ai>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,6 +64,16 @@ In addition to the required packages for the ROS2 gem, you will also need some a
...
+You can install it by running the following command
+```bash
+pip install python-statemachine
+```
+
+**Note:** Make sure you have the pip Python package manager installed.
+
 ## Installation ##
 
 - Use the [roscon_2022](https://github.com/aws-lumberyard-dev/o3de/tree/roscon_2022) branch of the `O3DE`.
```
Comment (michalpelka) [inline README.md:79]: I think that the address of this repo was updated recently.

**Review by michalpelka — COMMENTED** [review]
double check link

File: README.md (hunk 1 above)
Comment (alek-kam-robotec-ai) [reply README.md:79]: Fixed.

File: README.md [hunk 2]
```diff
@@ -82,43 +104,143 @@ colcon build --symlink-install
...
 
-1. For our scenario, spawn the following robots:
+1. Make sure that you have ROS2 workspace sourced
+
+```bash
+cd ~/o3de_kraken_ws
+source ./install/setup.bash
+```
+
+2. Make sure that you have `CycleDDS` as middleware
```
Comment (adamdbrw) [inline README.md:179]: Move point 1 and point 2 to a separate section in usage scenario - the first one: base steps (for every scenario). No need for repetition here

**Pull request merged**